docs/guides/1.18.0-upgrade-guide.md

```diff
@@ -8,10 +8,13 @@ The Terraform MongoDB Atlas Provider version 1.18.0 has a number of new and exci
 
 **New Resources, Data Sources, and Features:**
 
-- You can now scale the instance size and disk IOPS independently for each individual shard for sharded and geo-sharded clusters defined with `mongodbatlas_advanced_cluster. To learn more, see the [Migration Guide](advanced-cluster-new-sharding-schema). As part of these changes, added two new attributes:
+- You can now scale the instance size and disk IOPS independently for each individual shard for sharded and geo-sharded clusters defined with `mongodbatlas_advanced_cluster`. To learn more, see the [Migration Guide](advanced-cluster-new-sharding-schema). As part of these changes, added two new attributes:
     - Use the `replication_specs.*.zone_id` attribute in the `mongodbatlas_advanced_cluster` resource and data sources to specify the zone of each `replication_specs` object.
   - Use the `use_replication_spec_per_shard` attribute in the `mongodbatlas_advanced_cluster` data sources to specify whether to obtain `replication_specs` objects for each shard.
 
+- The cloud backup schedule for a cluster can now be used to configure copy settings by zone instead of replication object in each zone with the `mongodbatlas_cloud_backup_schedule` resource and data sources.  This also enables you to reference cluster zones using independent shard scaling, which no longer supports `replication_spec.*.id`.
```

Comment:
```suggestion
- The cloud backup schedule for a cluster can now be used to configure copy settings by zone instead of replication object in each zone with the `mongodbatlas_cloud_backup_schedule` resource and data sources.  This feature also enables you to reference cluster zones using independent shard scaling, which no longer supports `replication_spec.*.id`.
```